react-pool/src/utils/globalParameters.js

```diff
@@ -99,7 +116,7 @@ const LARGE_WIDGET_TRADING_LINK = window.reactInit.largeWidgetLink;
 // Global variables that are going to be used by the website
 
 // Set true or false if you wanna use the test-net (demo enviroment) or main-net enviroment of the pool node
-export const useTestNet = process.env.NODE_ENV === 'development'
+export const useTestNet = process.env.NODE_ENV === "development";
```

Comment:
This "testnet" thing can be misleading. People think they need this to run the pool on testnet. Please consider using something like "remoteNode" (not necessarily testnet).